importer.js

```diff
@@ -99,8 +99,8 @@ class Importer {
     var transformer = transform((record, callback) => {
       // TODO: add filter
       this.writeRecordToInflux(record)
-        .then(() => callback(null, '.'))
-        .catch((err) => {
+        .then(_ => callback(null, ''))
```

Comment:
please don't name vars as `_`. I know what you want to say, but I very often use https://lodash.com in projects. Very possible will add it later